src/views/Swap/components/CurrencyInputHeader.tsx

```diff
@@ -20,7 +30,11 @@ const CurrencyInputContainer = styled(Flex)`
   border-bottom: 1px solid ${({ theme }) => theme.colors.cardBorder};
 `
 
-const CurrencyInputHeader: React.FC<Props> = ({ title, subtitle, setIsChartDisplayed }) => {
+const ColoredIconButton = styled(IconButton)`
+  color: ${({ theme }) => theme.colors.textSubtle};
+`
```

Comment:
Does this affect anything? I think setting color on the icon below changes color already

Reply:
ChartDisable Icon use stroke that is not overrided by Svg 
https://github.com/pancakeswap/pancake-toolkit/blob/master/packages/pancake-uikit/src/components/Svg/Icons/ChartDisable.tsx#L8

maybe I should added in Svg component
`stroke: ${({ theme, color }) => getThemeValue(`colors.${color}`, color)(theme)};`